arcgis-ios-sdk-samples/Layers/Display WFS/DisplayWFSViewController.swift

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class DisplayWFSViewController: UIViewController {
+
+    @IBOutlet private weak var mapView: AGSMapView!
+    private var wfsFeatureTable : AGSWFSFeatureTable!
```

Comment:
```suggestion
    private var wfsFeatureTable: AGSWFSFeatureTable!
```

Please remove the space before the colon. It generates a warning for violating the [Colon](
https://github.com/realm/SwiftLint/blob/master/Rules.md#colon) linting rule.

![Screen Shot 2019-03-20 at 4 04 05 PM](https://user-images.githubusercontent.com/2257493/54724781-d2917080-4b29-11e9-8eeb-a1063e273309.png)